### PR TITLE
CAT-897 Added Deselect all Functionality

### PR DESCRIPTION
**Pull request by LauraBebek**
hopefully with no issues anymore :)

### CODE REVIEW

**Comment by avihric** [comment]
A test would be nice :smile:

**Comment by LauraBebek** [comment]
added and fixed tests!

File: catroid/src/org/catrobat/catroid/ui/fragment/ProjectsListFragment.java [hunk 1]
```diff
@@ -573,13 +573,26 @@ private void addSelectAllActionModeButton(ActionMode mode, Menu menu) {
...
+				} else {
+					adapter.clearCheckedProjects();
+					adapter.notifyDataSetChanged();
+					onProjectChecked();
+					selectAll = true;
+					selectAllView.setText(R.string.select_all);
 				}
-				adapter.notifyDataSetChanged();
-				onProjectChecked();
+
```
Comment (avihric) [inline catroid/src/org/catrobat/catroid/ui/fragment/ProjectsListFragment.java]: checkstyle does not like this

Comment (LauraBebek) [inline catroid/src/org/catrobat/catroid/ui/fragment/ProjectsListFragment.java]: check :)

**Comment by thmq** [comment]
https://jenkins.catrob.at/job/Catroid-Build-SourceTests-StaticChecks/3829/checkstyleResult/

**Pull request merged**